pkg/moov/disputes_test.go

```diff
@@ -61,3 +61,69 @@ func Test_GetDisputes_NotFound(t *testing.T) {
 
 	require.Equal(t, moov.StatusNotFound, httpErr.Status())
 }
+
+func Test_AcceptDisputes_Unauthorized(t *testing.T) {
+	mc := NewTestClient(t)
+
+	// We don't have any disputes to test against! So using a random id to accept will return unauthorized since we dont own the resource
```

Comment:
Can't we create a dispute in tests?

Reply:
Theres no endpoint to create a dispute on our docs https://docs.moov.io/api/money-movement/disputes/

and there are none existing in staging that i can test agains on myaccount. This comment was copied from the existing tests for disputes